src/pages/docs/projects/project-triggers/built-in-package-repository-triggers.md

```diff
@@ -15,21 +15,17 @@ We have renamed the feature and refreshed the UI, but it still works the same wa
 
 ## Getting started {#BuiltInPackageRepositoryTriggers-GettingStarted}
 
-If you use the [built-in Octopus package repository](/docs/packaging-applications/package-repositories), you can now select a package, that when uploaded it will automatically create a release.
+If you use the [built-in Octopus package repository](/docs/packaging-applications/package-repositories), you can now select a package, that when uploaded will automatically create a release.
 
 :::div{.warning}
 **Built-in repository only**
 
 This trigger only supports the [built-in package repository](/docs/packaging-applications/package-repositories). There is some support for external feeds using the [external feed triggers](/docs/projects/project-triggers/external-feed-triggers).
 :::
 
-From the project's trigger tab, click **ADD TRIGGER**, and then select the **Built-in package repository**:
+From the project's trigger tab, click **Add Trigger**, and then select **Built-in package repository**:
 
-:::figure
-![Add trigger dialog](/docs/projects/project-triggers/images/add-trigger-popup.png)
```

Comment:
The other trigger pages don't have a screenshot and they were easy to follow. Removing this to make them consistent.